src/ray/raylet/scheduling/scheduling_policy_test.cc

```diff
@@ -1,429 +0,0 @@
-    const auto to_schedule =
-        raylet_scheduling_policy::SchedulingPolicy(local_node, nodes)
-            .HybridPolicy(
-                req, 0.51, false, true, [](auto) { return true; }, true);
-    ASSERT_EQ(to_schedule, local_node);
-  }
-  {
-    // A CPU request can be be scheduled on a CPU node.
-    const ResourceRequest req = ResourceMapToResourceRequest({{"CPU", 1}}, false);
-    const auto to_schedule =
-        raylet_scheduling_policy::SchedulingPolicy(local_node, nodes)
-            .HybridPolicy(
-                req, 0.51, false, true, [](auto) { return true; }, true);
-    ASSERT_EQ(to_schedule, remote_node);
-  }
-  {
-    // A mixed CPU/GPU request should be scheduled on a GPU node.
-    const ResourceRequest req =
-        ResourceMapToResourceRequest({{"CPU", 1}, {"GPU", 1}}, false);
-    const auto to_schedule =
-        raylet_scheduling_policy::SchedulingPolicy(local_node, nodes)
-            .HybridPolicy(
-                req, 0.51, false, true, [](auto) { return true; }, true);
-    ASSERT_EQ(to_schedule, local_node);
-  }
-}
-
-TEST_F(SchedulingPolicyTest, SchedulenCPURequestsOnGPUNodeAsALastResort) {
-  // Schedule on remote node, even though the request is CPU only, because
-  // we can not schedule on CPU nodes.
-  ResourceRequest req = ResourceMapToResourceRequest({{"CPU", 1}}, false);
-  nodes.emplace(local_node, CreateNodeResources(0, 10, 0, 0, 0, 0));
-  nodes.emplace(remote_node, CreateNodeResources(1, 1, 0, 0, 1, 1));
-
-  const auto to_schedule =
-      raylet_scheduling_policy::SchedulingPolicy(local_node, nodes)
-          .HybridPolicy(
-              req, 0.51, false, true, [](auto) { return true; }, true);
-  ASSERT_EQ(to_schedule, remote_node);
-}
-
-TEST_F(SchedulingPolicyTest, ForceSpillbackTest) {
-  // The local node is available but disqualified.
-  ResourceRequest req = ResourceMapToResourceRequest({{"CPU", 1}, {"GPU", 1}}, false);
-
-  nodes.emplace(local_node, CreateNodeResources(2, 2, 0, 0, 1, 1));
-  nodes.emplace(remote_node, CreateNodeResources(0, 2, 0, 0, 0, 1));
-
-  auto to_schedule = raylet_scheduling_policy::SchedulingPolicy(local_node, nodes)
-                         .HybridPolicy(req, 0.51, true, false, [](auto) { return true; });
-  ASSERT_EQ(to_schedule, remote_node);
-}
-
-TEST_F(SchedulingPolicyTest, ForceSpillbackOnlyFeasibleLocallyTest) {
-  // The local node is better, but we force spillback, so we'll schedule on a non-local
-  // node anyways.
-  ResourceRequest req = ResourceMapToResourceRequest({{"CPU", 1}, {"GPU", 1}}, false);
-
-  nodes.emplace(local_node, CreateNodeResources(2, 2, 0, 0, 1, 1));
-  nodes.emplace(remote_node, CreateNodeResources(0, 2, 0, 0, 0, 0));
-
-  auto to_schedule = raylet_scheduling_policy::SchedulingPolicy(local_node, nodes)
-                         .HybridPolicy(req, 0.51, true, false, [](auto) { return true; });
-  ASSERT_TRUE(to_schedule.IsNil());
-}
-
-TEST_F(SchedulingPolicyTest, NonGpuNodePreferredSchedulingTest) {
```

Comment:
Is this moved somewhere else?

Reply:
It's moved to another file in a previous PR by @scv119. He probably forgot to remove this.